index.html

```diff
@@ -63,7 +64,7 @@ <h3 class="multi_story">Multi-Post Stories</h3>
   </div>
 
   <div class="recent_work_container flex">
- <div class="first_project">
+    <div class="first_project">
```

Comment:
- [ ]  i would suggest you please stick to the design as much as possible. The featured project image takes 60% of the page's width and is aligned to the left, the description takes the remaining 40% and is aligned to the right. There is equal padding from left and right. Please see the image below for reference.
- [ ] Kindly increase the font size of the placeholder of the `see-project` buttons.
### Your design
![his](https://user-images.githubusercontent.com/81727988/206658675-582144cb-39a5-4cb1-922c-a420c7d664ff.png)
### Figma design

![figma](https://user-images.githubusercontent.com/81727988/206658707-fb759443-2d93-4b53-adb0-c67da88445ec.png)